src/aiida/orm/nodes/data/code/abstract.py

```diff
@@ -381,3 +381,21 @@ def get_builder(self) -> 'ProcessBuilder':
         builder.code = self
 
         return builder
+
+    def _prepare_yml(self, main_file_name='', *args, **kwargs):
+        """Export code to a yaml file."""
+        import yaml
+
+        code_data = {}
+        sort = kwargs.get('sort', False)
+
+        for key in self.Model.model_fields.keys():
+            value = getattr(self, key).label if key == 'computer' else getattr(self, key)
+
+            # If the attribute is not set, for example ``with_mpi`` do not export it, because the YAML won't be valid
+            # for use in ``verdi code create`` since ``None`` is not a valid value on the CLI.
+            # TODO: Better handling of this should eventually be implemented in ``verdi code create``
```

Comment:
The annotation didn't relate to `computer`s specifically, rather to null-values in the YAML file which would result from keys not being set. In the past, when I tried this, having null-values in the YAML file would give an exception during `verdi code create`, so to avoid exporting a corrupted YAML file, better to not include these keys at all. However, I just tried it, and, apparently, now the user gets interactively prompted for the corresponding values, which is exactly the solution I'd have imagined, so I'll remove the TODO. I'd leave the check in the code, though, as I didn't add this, and I guess the original author had their reasons.